webview/src/experiments/components/App.test.tsx

```diff
@@ -348,13 +348,26 @@ describe('App', () => {
   })
 
   describe('Toggle experiment status', () => {
+    const getRowByLabel = (label: string) => {
+      const rows = screen.getAllByRole('row')
+
+      return rows.find(element => element.innerHTML.includes(label))
+    }
+
     it('should send a message to the extension to toggle an experiment when the row is clicked', () => {
       renderTable()
 
       const testClick = (label: string, id = label) => {
         mockPostMessage.mockReset()
 
-        fireEvent.click(screen.getByText(label))
```

Comment:
Since the label is no longer a toggle, we can't just get an element by the label text to check if things are working. I came up with a solution but it's probably over complicated and there is an easier way 😅